src/Connection.php

```diff
@@ -327,6 +327,11 @@ public function __call($method, $parameters)
         return $this->db->$method(...$parameters);
     }
 
+    public function getServerVersion(): string
+    {
+        return $this->db->command(['buildInfo' => 1])->toArray()[0]['version'];
```

Comment:
A couple of notes here:
* `buildInfo` is not part of the [stable API](https://www.mongodb.com/docs/manual/reference/stable-api/), so if the client was configured to use a stable API version this command will fail
* In a replica set, this will only return the version of the primary server; it may be worth noting that other servers in the replica set may be of a different version
* In a sharded deployment, the command is run on an arbitrary `mongos` from the discovered topology. This means that two subsequent runs of the `buildInfo` command could yield different results in a mixed deployment.

It's not necessarily critical, but it may be worth noting this in a comment. We can also consider marking the method as internal given the limitations.

Reply:
Added a comment to the method.